docs/BEMAN_LIBRAY_MATURITY_MODEL.md

```diff
@@ -0,0 +1,35 @@
+<!--
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+# The Beman Library Maturity Model
+
+This document specifies the Beman Library Maturity Model.
+Its goal is to create awareness about the current status of Beman libraries.
+
+## Introduction
+
+### Changing this document
+
+<!-- TODO: Maybe instead of duplicatind content from BEMAN_STANDARD.md#Changing-this-document here, we put a single section in root README.md -->
+
+## Beman library maturity model
+
+The Beman maturity model helps developers quickly assess the production readiness of Beman libraries by classifying them based on development phase and interface stability.
+
+<img src="./img/beman-library-maturity-model.png">
```

Comment:
Moving suggestion from Radu in a thread - "Could we generate logos as svg files?"

yes, this was also my plan or my question, but I forgot to ask.

Action/question 1: Do we want to do that?
Action 2: I will do that before final review/merging PR I need to us , but for now I would just to ask if you like the idea of these logos + a diagram to visually explain the flow.

As a side note, I did something quick to get something to show - https://new.express.adobe.com/id/urn:aaid:sc:EU:c9f3fd2b-ad26-488c-a92b-9cee6fca99d0?invite=true&promoid=Z2G1FQKR&mv=other - I'm not saying this is the best tool, just something I used for a PoC.

C

Reply:
Related to PNG vs SVG discussion, I posted at https://discourse.bemanproject.org/t/deciding-on-a-project-logo/102/15.

As a side node: this is a style related issue. If all other major issues with this PR will be resolved, I will add SVG version of the logos in a follow-up PR.

Reply:
https://github.com/bemanproject/beman/issues/81 - linked issue.